xc/xc7/tests/soc/ibex/ibex.patch

```diff
@@ -0,0 +1,112 @@
+diff --git a/examples/fpga/artya7/rtl/top_artya7.sv b/examples/fpga/artya7/rtl/top_artya7.sv
+index 37304b4..c1a0b05 100644
+--- a/examples/fpga/artya7/rtl/top_artya7.sv
++++ b/examples/fpga/artya7/rtl/top_artya7.sv
+@@ -11,7 +11,7 @@ module top_artya7 (
+   parameter int          MEM_SIZE     = 64 * 1024; // 64 kB
+   parameter logic [31:0] MEM_START    = 32'h00000000;
+   parameter logic [31:0] MEM_MASK     = MEM_SIZE-1;
+-  parameter              SRAMInitFile = "";
++  parameter              SRAMInitFile = "led.vmem";
+ 
+   logic clk_sys, rst_sys_n;
+ 
+diff --git a/shared/rtl/fpga/xilinx/clkgen_xil7series.sv b/shared/rtl/fpga/xilinx/clkgen_xil7series.sv
+index e41b4b0..5ca5d6c 100644
+--- a/shared/rtl/fpga/xilinx/clkgen_xil7series.sv
++++ b/shared/rtl/fpga/xilinx/clkgen_xil7series.sv
+@@ -27,10 +27,10 @@ module clkgen_xil7series (
+     .STARTUP_WAIT         ("FALSE"),
+     .DIVCLK_DIVIDE        (1),
+     .CLKFBOUT_MULT        (12),
+-    .CLKFBOUT_PHASE       (0.000),
++    .CLKFBOUT_PHASE       (0),
+     .CLKOUT0_DIVIDE       (24),
+-    .CLKOUT0_PHASE        (0.000),
+-    .CLKOUT0_DUTY_CYCLE   (0.500)
++    .CLKOUT0_PHASE        (0),
++    .CLKOUT0_DUTY_CYCLE   (500)
```

Comment:
@rw1nkler I don't remember why but in SymbiFlow `*_PHASE` should be multiplied by 1000 but `*_DUTY_CYCLE` should be multiplied by **100000**. So to get 50% duty cycle it has to be either changed to 50000 or the parameter should not be given at all (then it defaults to 50% duty).

Reply:
This is no longer needed after the changes introduced in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1640